examples/xyz.js

```diff
@@ -1,41 +1,15 @@
-goog.require('ol.Attribution');
 goog.require('ol.Map');
 goog.require('ol.View');
-goog.require('ol.control');
 goog.require('ol.layer.Tile');
-goog.require('ol.source.OSM');
 goog.require('ol.source.XYZ');
 
 
-var attribution = new ol.Attribution({
-  html: 'Tiles &copy; <a href="http://maps.nls.uk/townplans/glasgow_1.html">' +
-      'National Library of Scotland</a>'
-});
-
```

Comment:
I didn't want to 'debate every change headed in this direction', not at all. In this particular case I think it was a misreading of the diff. I didn't realize that the source is gone as well.

I am +1 on example simplifications and that they concentrate one feature at a time.